.jhipster/UserDetails.json

```diff
@@ -11,7 +11,7 @@
       "fieldType": "String"
     },
     {
-      "fieldName": "company",
+      "fieldName": "companyName",
```

Comment:
We need to change the company String filed to companyName so it would not conflict with the company table,